content/ko/docs/concepts/overview/kubernetes-api.md

```diff
@@ -93,19 +93,22 @@ kube-apiserver 구성 요소에
 `/openapi/v3` 디스커버리 엔드포인트는 사용 가능한 모든 
 그룹/버전의 목록을 제공한다. 이 엔드포인트는 JSON 만을 반환한다.
 이러한 그룹/버전은 다음과 같은 형식으로 제공된다.
-```json
+
+```yaml
 {
     "paths": {
-        ...
+        ...,
         "api/v1": {
             "serverRelativeURL": "/openapi/v3/api/v1?hash=CC0E9BFD992D8C59AEC98A1E2336F899E8318D3CF4C68944C3DEC640AF5AB52D864AC50DAA8D145B3494F75FA3CFF939FCBDDA431DAD3CA79738B297795818CF"
         },
         "apis/admissionregistration.k8s.io/v1": {
             "serverRelativeURL": "/openapi/v3/apis/admissionregistration.k8s.io/v1?hash=E19CC93A116982CE5422FC42B590A8AFAD92CDE9AE4D59B5CAAD568F083AD07946E6CB5817531680BCE6E215C16973CD39003B0425F3477CFD854E89A9DB6597"
         },
-        ...
+        ....
+    }
 }
 ```
+<!-- for editors: intionally use yaml instead of json here, to prevent syntax highlight error. -->
```

Comment:
오타가 있는데, upstream (영문) 에도 이렇게 되어 있네요.. ㅎㅎ

```suggestion
<!-- for editors: intentionally use yaml instead of json here, to prevent syntax highlight error. -->
```

Reply:
이거 제가 고치겠습니다..!